diff --git a/ios/ExtensionStorage.podspec b/ios/ExtensionStorage.podspec
index ba12b6c8e0906c4fc146b62061314203f91c63b8..c65b18993e7e64e01c406c84f1ed6ce633f7cc83 100644
--- a/ios/ExtensionStorage.podspec
+++ b/ios/ExtensionStorage.podspec
@@ -5,7 +5,7 @@ Pod::Spec.new do |s|
   s.description    = 'A sample project description'
   s.author         = ''
   s.homepage       = 'https://docs.expo.dev/modules/'
-  s.platform       = :ios, '15.1'
+  s.platform       = :ios, '16.4'
   s.source         = { git: '' }
   s.static_framework = true
 
